CMakeLists.txt

```diff
@@ -70,52 +64,57 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# Set the minimum python version to 3.7
+set(PYBIND11_PYTHON_VERSION 3.7)
```

Comment:
Not really specific to this PR, but The readme isn't saying anything regarding which minor revision of Python3 we support. Some colleagues ran into trouble trying to instal SP3 with python 3.6, although I think it is supposed to be possible. Maybe we should specify 3.7+ in the readme too

Reply:
We talked about it about 2-3 weeks ago on gitter, we could allow any 3.X versions, but warn the users that the exceptions raised in python scrips won't be intercepted by SOFA (or something like that), @damienmarchal  knows more about this. That way we are a little bit less restrictive as a lot of LTS distributions run on python 3.5 or 3.6

We should do it in another PR tho as it will be more clear in the git history.